Controller/Oauth/Index.php

```diff
@@ -91,6 +96,7 @@ public function __construct(
         $this->nostoHelperScope = $nostoHelperScope;
         $this->nostoHelperCache = $nostoHelperCache;
         $this->storeRepository = $storeRepository;
+        $this->storeManager = $storeManager;
```

Comment:
Do we need this? We already have `nostoHelperScope ` and you can get the store from there. https://github.com/Nosto/nosto-magento2/blob/09af2c2a690edf805c7d14924d3d6e180d22eb9a/Controller/Oauth/Index.php#L159 We need to ditch either the scope helper or the store manager